examples/sample-app/cleanup.sh

```diff
@@ -3,8 +3,13 @@
 echo "Killing openshift all-in-one server ..."
 pkill -x openshift
 
+echo "Stopping all k8s docker containers on host ..."
+docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
```

Comment:
thanks, got it,  but i still worry about that the command will find out all containers not only include  'k8s_'. This is a potential risk. And if we filter the outputs by prefix 'k8s_', it seems not a better change compare to the origin implemention. How do you think?